docs/manuals/core/entities/data-node-mgt.md

```diff
@@ -165,4 +165,27 @@ writes it on the data node:
     data_retrieved = data_node.read()
     ```
 
+# Get parent scenarios, pipelines and tasks
+
+To get the parent entities of a data node, like the tasks that use the data node, or the pipeline that includes the tasks, or the scenario that contains the pipeline, you can use the `DataNode.get_parents()^` method. The method will return a dictionary with the key being the level of the parent ("scenarios", "pipelines" or "tasks") and the value is a set of the parent entities corresponding to the level. You can also use `taipy.get_parents()^`, which takes a child entity such as the data node as input, to achieve the same result.
+
+!!! Example
+
+    ```python linenums="1"
+    import taipy as tp
+    import my_config
+
+    # Creating a scenario from a config
+    scenario = tp.create_scenario(my_config.monthly_scenario_cfg)
+
+    # Retrieve a data node
+    data_node = scenario.sales_history
+
+    # get the parent entities of the data node
+    parent_entities = data_node.get_parents()
+
+    # get the parent entities of the data node
+    tp.get_parents(data_node)
```

Comment:
Can you print `data_node.get_parents()` and `tp.get_parents(data_node)` and provide the output you got ? That would help the user understand the returned format.
Thx

Reply:
Sure!